### DISCUSSION TITLE
Failed to reload dynamic route

### DISCUSSION THREAD

**Discussion by robin-gustafsson** [wpengine/faustjs#509] [Q&A]
Hi! 

I want to use usePage and usePost on same route but I got this error: _"Failed to reload dynamic routes: Error: You cannot use different slug names for the same dynamic path ('postSlug' !== 'pageSlug')"_

**Comment by josephfusco** [comment]
Hey @Robin-Gustafsson, we don't plan on supporting this right now however we're open to discussion on this. Do you mind providing more context to your use case?
